packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -135,22 +135,33 @@ export const DevServerMixin = {
     _setupExtensions(app, options) {
         logger.info('Setting up extensions...')
 
-        // TODO: support extensions options array syntax i.e. ['extension-a', {}]
-        const extensions = options.mobify?.app?.extensions || []
+        // Normalize the extensions list for easier parsing
+        const extensions = (options.mobify?.app?.extensions || [])
+            .map((extension) => {
+                return {
+                    name: Array.isArray(extension) ? extension[0] : extension,
```

Comment:
There is a utility function called 'expand' in the `resolver-utils.js` file that might be something you can use here.

Reply:
Update: as we discussed, we won't call `expand` now. But later we'll consider calling it, after it's been updated/refactored.